packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -319,12 +323,19 @@ export default class Reachability {
           return targetMember;
         },
         set: (target, property, value) => {
-          // only intercept elapsed property
-          if (property === ELAPSED) {
-            // @ts-ignore
-            resolve({clusterId: peerConnection.key, elapsed: value});
-
-            return true;
+          switch (property) {
+            case ELAPSED:
+              // @ts-ignore
+              resolve({clusterId: peerConnection.key, elapsed: value});
+
+              return true;
+            case CLIENT_MEDIA_IP:
+              // @ts-ignore
+              resolve({clusterId: peerConnection.key, clientMediaIP: value});
```

Comment:
I don't think this is needed. The promise is resolved when the "elapsed" property is set, so with the current way you have the code calling setClientMediaIPs() after setLatencyAndClose() then this line will do nothing, because the promise is already resolved. I think you can just leave this code block as it was and only resolve the promise when "elapsed" is set and don't do anything specials for CLIENT_MEDIA_IP and just make sure that setLatencyAndClose() is the last thing that is called.

Reply:
reverted, and made sure `setLatencyAndClose ` is last

Reply:
but how will the value for clientMediaIP get through the promise in the end? if i revert this back, it won't be part of the iceResults and won't be able to add it to the reachabilityResult.

Reply:
you need to store it somehow and then include it in the resolve() call that's done in this line: https://github.com/webex/webex-js-sdk/blob/master/packages/%40webex/plugin-meetings/src/reachability/index.ts#L325
the current code stores the elapsed property on the peer connection object itself and I see you're doing the same storing it as `publicIPs` - I guess it's fine - I'm not a big fan of that approach but it's consistent with the existing code in this module.